en/frames_multicopter/holybro_x500v2_pixhawk6c.md

```diff
@@ -0,0 +1,159 @@
+# Holybro X500 V2 + Pixhawk 6C (PX4 Dev Kit)
+
+This topic provides full instructions for building the [Holybro X500 V2 ARF Kit](http://shop.holybro.com/x500-v2-kit_p1288.html), also known as the Holybro PX4 Dev Kit.
+
+![The fully built vehicle with props removed](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/kit_no_props.jpg)
+
+
+## Assembly
+
+### Holders
+
+**Screw**-  Sunk Screw M2.5*6 12pcs
+
+1. Insert the hanger rubber ring gasket in each of their respective hangers.
+   Do not use sharp objects to press the rubbers inside.
+
+   @[youtube](https://youtu.be/4Tid-FCP_aI)
+
+
+1. Take the battery mounting board and screw it with the slide bar clip using the Sunk Screw M2.5*6.
+
+   @[youtube](https://youtu.be/9E-rld6tPWQ)
```

Comment:
For every image, please replace these with a "final" image that you can click to go to the youtube. 

The syntax will end up something like this:
```
[![](this_is_the_image_in_assets_directory)](https://youtu.be/9E-rld6tPWQ))
```

Don't get me wrong, I LOVE the videos. But in plenty of places they will be slow to load, if they are able to load at all. So what you want here is an image that shows the final state you want people to get to - in other words, if they don't see the video they can STILL use the document. 

Make sense? The images should be in the assets directory.

Reply:
PS, here is a preview of how the links would behave: https://hamishwillee.github.io/multidocsify_test/#/redir_repo/en/akshata-01/PX4-user_guide/dev_guides/frames_multicopter/holybro_x500v2_pixhawk6c?id=power-module

Though note, the images are autogenerated in this case - I don't want to do that because the images are the initial image in the file, not the "final thing you want to produce"

Reply:
Got it, makes sense.

Reply:
@hamishwillee  
Since I do not have images, does a screenshot of the final image of each video work? 

Should I crop the part where YouTube's play & pause is seen? 
![image](https://user-images.githubusercontent.com/108534583/199400157-c9885953-9d59-4144-acbe-0a291bb39e20.png)